client/src/components/StreamField/blocks/BaseSequenceBlock.js

```diff
@@ -580,6 +588,22 @@ export class BaseSequenceBlock {
     this.blockCountChanged();
   }
 
+  initDragNDrop() {
+    this.sortable = Sortable.create(this.sequenceContainer.get(0), {
+      handle: '[data-streamfield-action="DRAG"]',
+      animation: 200,
+      // Only drag blocks, not insertion controls.
+      draggable: '[data-streamfield-child]',
+      onEnd: (e) => {
+        // Only consider StreamField blocks, not the insertion controls.
+        const { oldDraggableIndex, newDraggableIndex } = e;
+        if (oldDraggableIndex !== newDraggableIndex) {
+          this.moveBlock(oldDraggableIndex, newDraggableIndex);
+        }
```

Comment:
I’ve been hesitant to add tests for this as this is essentially the same as `moveBlock` but with specific identifiers. I’m not feeling too good about simulating drag events in unit tests just to see that `moveBlock` is called with correct identifiers, this would feel a bit like testing Sortable.js internals. But perhaps there is room to write more tests for `moveBlock`?

Reply:
If I were to write a test for this, I'd probably ensure that when the drag ends (or when the event is dispatched), `moveBlock` is called with the right arguments. Then, test `moveBlock` by itself (if there aren't tests for it already)